contract/src/main/java/com/ritense/valtimo/contract/security/jwt/provider/SecretKeyProvider.java

```diff
@@ -24,6 +24,6 @@ public interface SecretKeyProvider {
 
     boolean supports(SignatureAlgorithm algorithm, Claims claims);
 
-    Key getKey(SignatureAlgorithm algorithm);
+    Key getKey(SignatureAlgorithm algorithm, String kId);
```

Comment:
Looks like a breaking change. Can you implement it in such a way that it's non-breaking?

Reply:
no, I am afraid not. KeyID is necessary to get the correct key from the keyset.